_search-plugins/neural-sparse-search.md

```diff
@@ -390,4 +390,15 @@ The response contains both documents:
 
 ## Next steps
 
-- To learn more about splitting long text into passages for neural search, see [Text chunking]({{site.url}}{{site.baseurl}}/search-plugins/text-chunking/).
\ No newline at end of file
+- To learn more about splitting long text into passages for neural search, see [Text chunking]({{site.url}}{{site.baseurl}}/search-plugins/text-chunking/).
+
+## FAQ
+
+1. When using connectors to call remote service like SageMaker, the ingestion/search fails due to remote connector throttling exception. How to mitigate this?
+
+- Please modify the connector [`client_config`]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/blueprints/#configuration-parameters) to decrease the `max_connection` to prevent max concurrent connection exceeds the threshold of remote service. You can also modify the retry settings to flatten the request spike during ingestion.
```

Comment:
Not sure whether this is the correct way to link other documents. Please help correct it if I make mistakes

Reply:
This correct @zhichao-aws